### PR TITLE
Fix 128 byte mode hvx

### PR DESCRIPTION
**Pull request by pranavb-ca**

### CODE REVIEW

**Comment by pranavb-ca** [comment]
@dsharletg

File: src/CodeGen_Hexagon.cpp [hunk 1]
```diff
@@ -62,11 +62,6 @@ std::unique_ptr<llvm::Module> CodeGen_Hexagon::compile(const Module &module) {
         if (module.target().has_feature(Halide::Target::HVX_64))
```
Comment (dsharletg) [inline src/CodeGen_Hexagon.cpp]: The hvx_setting declared/set a few lines up from here is now un-used, can you remove it?

Comment (dsharletg) [inline src/CodeGen_Hexagon.cpp]: Also, this 'if' should probably just be:

    if (module.target().has_feature(...) && module.target().has_feature(...))

or even

    if (module.target().features_all_of({Target::HVX_64, Target::HVX_128}))

**Comment by dsharletg** [comment]
This method of passing hvx-double to LLVM looks a *LOT* better to me, thanks for fixing it this way.

**Comment by pranavb-ca** [comment]
@dsharletg : Done.

**Pull request merged**